common/pb/event_notify_user.proto

```diff
@@ -5,6 +5,7 @@ message Event_NotifyUser {
     enum NotificationType {
         PROMOTED = 1;
         WARNING = 2;
+        IDLEWARNING = 3;
```

Comment:
To avoid the problem of the message defaulting to "PROMOTED", you could set an explicit default value for NotificationType, see https://developers.google.com/protocol-buffers/docs/proto#optional
Right now since this is an `enum`, it defaults to its first value.